### ISSUE TITLE
Export to PDF/Word

### ISSUE THREAD

**Issue by marcel1334** [OHDSI/WhiteRabbit#167]
Instead of the huge Excel sheet another way to show the results. Maybe generate a word document/PDF including histograms, boxplots (where possible) and other summary information (number of attributes, records, etc) .

**Comment by PRijnbeek** [comment]
Need to understand this a bit better maybe with an example.

RaH will read the file and will show this in a more user-friendly way (which could be extended). What is it exactly that you are missing?  What problem does it solve to create histograms when your are designing the ETL document? etc

**Comment by PRijnbeek** [comment]
One reason i could think of is if we would like to have a repository of standardised source schema's etc auto generated from the scan as they do in https://dataedo.com

(good to watch their video to get some ideas, i do link that it is generates a ERD etc from the database and allows the annotation as we talked about in other issues, it also generates a html report that you can share)

I am not sure data custodians would like to share this though?

**Comment by marcel1334** [comment]
Has overlap with thew discussion in issue #165 

Looks like a great tool to document the source database. Can be exported and maybe this info can be imported up by RabbitInAHat next to the WR scan report.

It's not that they have to share this with the whole community, but very usefull to share with the SME doing the ETL.

**Comment by MaximMoinat** [comment]
> RaH will read the file and will show this in a more user-friendly way (which could be extended).

Agree, I like the principle that all output from WR is to be ingested/visualised by RiaH. So the WR development should be driven by the following question: **what do we want to see in RiaH to inform our ETL decisions**? 

If it is not useful in RiaH, then we are just creating a profiling tool, which is not the main scope of WR.

**Comment by marcel1334** [comment]
Agree, the scope is NOT to create a full featuring profiling tool, but only a couple of extra and simple stats to give an better idea of the table and help the ETL design. It has to be added value for the ETL design phase.
